### DISCUSSION TITLE
Add a way to customize issues/pr's notifications

### DISCUSSION THREAD

**Discussion by soredake** [community/community#52799] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

In the web interface you can customize your notifications and subscribe to only closed event for example, it will be nice if there is a way to customize notifications for issues/pr's in github app.

![image](https://user-images.githubusercontent.com/5204968/232008999-a2dae154-891b-44eb-97a9-1ff38fa9350f.png)
![image](https://user-images.githubusercontent.com/5204968/232009035-b078a880-2edf-4ae2-afee-7cce9749a45d.png)

**Comment by stevepopovich** [comment]
Hi @soredake these settings should affect the notifications you get in the mobile app. Is that not happening for you? Or are you looking for more custom options that are specific to mobile

**Reply by soredake** [reply]
@soredake there is no way to customized notifications in app for android like in web version, you can only subscribe or unsubscribe.

**Reply by stevepopovich** [reply]
Oh so you just want these settings in the mobile app?

**Reply by soredake** [reply, marked as answer]
yes.

**Reply by stevepopovich** [reply]
I will pass this on to the team for consideration!

**Reply by asl29** [reply]
> Hi @soredake these settings should affect the notifications you get in the mobile app. Is that not happening for you? Or are you looking for more custom options that are specific to mobile

**Comment by mrwan11** [comment]
The new

**Comment by Anouomgharbivsseagjdzzvz** [comment]
Qweqwe4844

**Comment by soredake** [comment]
![Screenshot_20240102-101121_GitHub](https://github.com/community/community/assets/5204968/947c82cc-0053-45e4-a355-b401fa7bfd3e)

For repositories, there are already settings for custom notifications.
